src/orchestration/KubernetesClient.js

```diff
@@ -40,6 +40,27 @@ class KubernetesClient {
    * @returns {Promise<EngineContainerSpec[]>} A promise to a list of engine container specs.
    */
   static async listEngines() {
+    const replicaMap = new Map();
+    try {
+      const replicaSets = await kubeHttpGet('/apis/apps/v1/replicasets');
+      replicaSets.items.forEach((item) => {
+        replicaMap.set(item.metadata.uid, item);
+      });
+    } catch (error) {
+      logger.warn('Could not retrive replicaset information, verify your RBAC settings');
+    }
+
+    const deploymentMap = new Map();
+    try {
+      const deployments = await kubeHttpGet('/apis/apps/v1/deployments');
+      deployments.items.forEach((item) => {
+        deploymentMap.set(item.metadata.uid, item);
+      });
+    } catch (error) {
+      logger.warn('Could not retrive deployment information, verify your RBAC settings');
+    }
+
+
```

Comment:
Does not lint complain about this extra empty line?

Reply:
Nope, but I removed it anyways ;)